tests/utils/browser.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const feedBackDocs = async (testName = 'allLogs') => {
+  const feedBackDocs = await browser.executeAsync(feedBackDocsScript).flat();
```

Comment:
I think the `flat` should be part of a new statement otherwise it may be called on the Promise instead of the array? ie:

```suggestion
  const feedBackDocs = await browser.executeAsync(feedBackDocsScript);
  const flattened = feedBackDocs.flat();
```